components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/PasswordRecoveryApi.java

```diff
@@ -245,6 +235,7 @@ public InitiateAllQuestionResponse securityQuestionsGet(String username, String
      * @param resetPasswordRequest key, password and optional metadata properties (required)
      * @throws ApiException if fails to make API call
      */
+    @Deprecated
```

Comment:
Not used ?

Reply:
No. I checked in the accountrecoveryendpoint. This API has been used as an alternative. https://github.com/wso2/carbon-identity-framework/blob/70741561b83a981ba9cc5641a791f0a4a2619158/components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/NotificationApi.java#L208

Reply:
https://github.com/wso2/identity-apps/blob/8de1bc2d7e21694046b863e91a66b80ee61bdcb6/apps/recovery-portal/src/main/webapp/password-reset-complete.jsp#L84